File: ejercicio69 ContarDigitosYSumar/java/ejercicio69.java
import java.util.Scanner;

public class ContarDigitosYSumar {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Ingrese un número: ");
        int numero = scanner.nextInt();

        int suma = 0;
        int cantidadDigitos = 0;

        while (numero > 0) {
            int digito = numero % 10;
            suma += digito;
            cantidadDigitos++;
            numero /= 10;
        }

        System.out.println("Cantidad de dígitos: " + cantidadDigitos);
        System.out.println("Suma de los dígitos: " + suma);

        scanner.close();
    }
}

File: Ejercicio 69/ejercicio69.py
numero = int(input("Ingrese un número: "))

suma = 0
cantidad_digitos = 0

while numero > 0:
    digito = numero % 10
    suma += digito
    cantidad_digitos += 1
    numero //= 10

print("Cantidad de dígitos:", cantidad_digitos)
print("Suma de los dígitos:", suma)
